src/styles/theme-overrides/theme-overrides.scss

```diff
@@ -3,12 +3,19 @@
 /* GENERAL */
 $theme-image-path: "~uswds/src/img";
 
+/* COLOR */
+$theme-text-color: $mocto__blue-design-token;
```

Comment:
Good idea to use a variable (like $mocto__blue-design-token) and set the theme color equal to that variable!